stitches.config.ts

```diff
@@ -0,0 +1,23 @@
+import { createStitches } from '@stitches/react';
+import { colorPrimitives } from './src/primitives/colorPrimitives';
+
+export const {
+  styled,
+  css,
+  globalCss,
+  keyframes,
+  getCssText,
+  theme,
+  createTheme,
+  config,
+} = createStitches({
+  theme: {
+    colors: colorPrimitives,
+  },
+  media: {
+    bp1: '(min-width: 480px)',
```

Comment:
Do we know where this breakpoint value comes from? One thing to note is that this `media` object can be expanded when we work on parts of the UI kit that require responsive style. I'm not opposed to omitting the breakpoints configuration from this PR, and adding it in when we need

Reply:
This was just placeholder content, I think it came from the stitches docs somewhere. Totally get that its confusing being there, I've now removed the placeholder content and replaced with stubs to make it clearer 😄

Reply:
We want our stitches theme to align with our current "theme" which follows. See the `theme` export (found in theme.ts) for more info.